samples/21.react-media-template/src/components/InkCanvas.jsx

```diff
@@ -20,8 +21,20 @@ export const InkCanvas = ({ isEnabled, liveCanvas }) => {
     }
   };
 
+  const setToHighlighter = useCallback(() => {
```

Comment:
This callback isn't necessary. It's only called from the `useEffect` below that already has logic that uses the `inkingManager`. You can just call `inkingManager.tool = InkingTool.highlighter;` from inside the `useEffect`.